Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
@DomT4 @ilovezfs I just upgraded `npm` inplace using `npm update -g` and within the current `node 7.10.0` bottle. Since I use it for absolutely nothing, feel free to feed me commands to check how broken this may have made things. I can't really do it on my own because I know next to nothing about `npm`.

> Here's another question. Let's say we go with the old npm for now. What happens when someone tries to upgrade npm in place at that point?